### ISSUE TITLE
[pt] localization content/pt/docs/languages/js/_index.md

### ISSUE THREAD

**Issue by leandrocarpenter** [open-telemetry/opentelemetry.io#5323]
* Add pt location to content/pt/docs/languages/js/_index.md

**Comment by svrnm** [comment]
@leandrocarpenter do you want to contribute this page? cc @open-telemetry/docs-pt-approvers

**Comment by leandrocarpenter** [comment]
Yes, I create this PR -> https://github.com/open-telemetry/opentelemetry.io/pull/5415
